scripts/airgapped/requirements.txt

```diff
@@ -1,4 +1,5 @@
 docker
-#FIXME: remove requests pin when https://github.com/docker/docker-py/issues/3256 is solved 
+#FIXME: remove requests pin when https://github.com/docker/docker-py/issues/3256 is solved
```

Comment:
the issue linked is closed, can we remove this now?

Reply:
Yep. I also confirmed docker+requests is working with the following commands
```bash
python3 scripts/get-all-images.py releases/1.9/stable/bundle.yaml > images.txt
# edit images.txt to include 2 images
python3 scripts/airgapped/save-images-to-cache.py images.txt
```

fd6394e